freqtrade/rpc/rpc.py

```diff
@@ -97,9 +97,13 @@ def _rpc_trade_status(self) -> List[Dict[str, Any]]:
                     current_rate = self._freqtrade.get_sell_rate(trade.pair, False)
                 except DependencyException:
                     current_rate = NAN
+
                 current_profit = trade.calc_profit_percent(current_rate)
                 fmt_close_profit = (f'{round(trade.close_profit * 100, 2):.2f}%'
                                     if trade.close_profit else None)
+                sl_percentage = round(((trade.stop_loss - current_rate) / current_rate) * 100, 2)
```

Comment:
i know it's still WIP - but don't do this.

a) the calculation is wrong - current_rate is only correct right after opening - or in case of trailing stop WHEN the current_rate is higher than the open rate.
b) just don't ...

we should store the percentage value in the database as well - should be easy to do (look for `adjust_stop_loss`) and saves you calculation repetitions which are meant to fail

Reply:
you are right, I'll remove it from this PR then we deal with the % in another PR in DB ?